Sources/TokamakDOM/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Tokamak App</title>
+    <script>
+      window.Tokamak = {
```

Comment:
Is there any benefit to having this snippet written in JS instead of Swift? Performance benefits for example?

Reply:
@carson-katri I assume you've addressed this with your benchmarks comment? Could still benefit with a code comment either making a reference to benchmarks or some other explanation.

Reply:
I actually wasn't able to test that. For now, I've rewritten that in Swift. It definitely simplifies things a bit.